### PR TITLE
Added link to C++17 Standard Draft in paragraphs.

### PR DESCRIPTION
**Pull request by wrazik**
This is fix for https://github.com/knatten/cppquiz/issues/152 .

### CODE REVIEW

**Comment by knatten** [comment]
Thank you so much! I'll review this in a few days.

**Comment by knatten** [comment]
Looks very good!

I think we should insert links also with missing paragraph numbers, what do you think? Currently  `[over.best.ics]§16.3.3.1¶2.2` gets a link, but ` [over.best.ics]§16.3.3.1` does not, but probably should?

**Comment by wrazik** [comment]
@knatten Mentioned link should point to: https://timsong-cpp.github.io/cppwp/n4659/over.best.ics ?

File: cppquiz/quiz/templatetags/quiz_extras.py [hunk 1]
```diff
@@ -7,12 +7,27 @@
 
 register = template.Library()
 
+def format_paragraph(paragraph_name, paragraph_number, full_text):
```
Comment (knatten) [inline cppquiz/quiz/templatetags/quiz_extras.py]: Minor nitpick: I'm a bit unsure about the names here. I think maybe `paragraph_name` should be `section_name`? 

Actually not sure about these names myself, so I asked the Twitterverse, let's see if we get any feedback: https://twitter.com/knatten/status/1132974180104851456

**Comment by knatten** [comment]
Yes, that was what I was thinking. I know we have a few of those.

**Comment by wrazik** [comment]
Suggestions applied! Please let me know if something needs to be fixed.

**Comment by DrChr** [comment]
I'd be happy to help by testing, so if/when you'd like me to test just let me know.
cheers,
Christian

**Comment by knatten** [comment]
Looks good, thanks a lot! Could you please squash the two commits? While you're at it there are two pieces of trailing white space which would be good to remove too (line 13 and 30).

And @DrChr thanks for offering to help with testing! I've deployed the PR to http://beta.cppquiz.org/ now.

**Comment by wrazik** [comment]
@knatten Done

**Comment by wrazik** [comment]
@knatten I'm not sure what is happening, but not all questions are linked, please take a look:
http://beta.cppquiz.org/quiz/question/131?result=OK&answer=id&did_answer=Answer

**Comment by knatten** [comment]
Comparing it to the [original code](http://cppquiz.org/quiz/question/131?result=OK&answer=id&did_answer=Answer) it's also lost italics. I don't have time to properly dig into this right now, but could it be that the previous code matched all references, but that the new one only matches the first?

**Comment by wrazik** [comment]
Yeah, it may be it. I will add a proper test for it.

**Comment by wrazik** [comment]
@knatten Correction ready, I didn't squash yet, I don't want you to check same code again :) Changes are here: https://github.com/knatten/cppquiz/pull/155/commits/0c8efd5a8e8c76b76ba15077543c583fb2929c4a

**Comment by knatten** [comment]
Deployed your new code to the beta site now, looks much better! http://beta.cppquiz.org/quiz/question/131?result=OK&answer=id&did_answer=Answer

Please squash to one commit, and I'll merge! (And if you want to be super tidy, remove the trailing whitespace in your new test.)

**Comment by wrazik** [comment]
Done! Thanks for the support.

**Comment by knatten** [comment]
Great, thanks a lot for your contribution! :)

**Pull request merged**